tests/target/issue_1255.rs

```diff
@@ -0,0 +1,8 @@
+#![feature(specialization)]
+
+trait Trait {
+    type Type;
+}
+impl<T> Trait for T {
+    default type Type = u64;
+}
```

Comment:
These files are slightly different in that the source version has a terminating newline but the target one does not. That may cause problems in the future.

However, if you want just an idempotence test, then you don't need the source version at, just a target file is necessary.

Could you add a comment to the file to explain what it being tested please?